### DISCUSSION TITLE
TypeScript errors in web editor

### DISCUSSION THREAD

**Discussion by Hypnosphi** [community/community#5356] [Codespaces]
Is there any way to view TypeScript errors in the free Web-based editor?

**Comment by bamurtaugh** [comment]
Hi @Hypnosphi - you should experience partial IntelliSense mode while in the web, which you can read more about here: https://code.visualstudio.com/docs/nodejs/working-with-javascript#_partial-intellisense-mode. Please let me know if you have any questions or had something else in mind.
